octodns_scaleway/__init__.py

```diff
@@ -80,19 +98,37 @@ def zone_records(self, zone_name):
         except ScalewayClientForbidden:
             return []
 
+    def zone_ds_records(self, domain):
+        try:
+            return self._request('GET', f'/domains/{domain}'
+                                 ).json()['dnssec']['ds_records']
+        except ScalewayClientForbidden:
+            return []
+
     def record_updates(self, zone_name, data):
         self.log.debug(f'record_updates: zone_name={zone_name}, data={data}')
         self._request('PATCH', f'/dns-zones/{zone_name}/records',
                       data=data)
 
+    def dnssec_enable(self, domain, data):
+        self.log.debug(f'dnssec_enable: domain={domain}, data={data}')
+        self._request('POST', f'/domains/{domain}/enable-dnssec',
+                      data=data)
+
+    def dnssec_disable(self, domain):
+        self.log.debug(f'dnssec_disable: domain={domain}')
+        self._request('POST', f'/domains/{domain}/disable-dnssec')
+
 
 class ScalewayProvider(BaseProvider):
     SUPPORTS_GEO = False
     SUPPORTS_DYNAMIC = True
+    SUPPORTS_ROOT_NS = True
```

Comment:
I've added support for root NS
https://github.com/octodns/octodns/blob/main/octodns/source/base.py#L9